README.md

```diff
@@ -78,6 +80,29 @@ for submitting pull requests to us.
 
 Visit [Contribute](https://github.com/PowerGridModel/power-grid-model/contribute) for a list of good first issues in this repo.
 
+## Citations
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8054429.svg)](https://zenodo.org/record/8054429).
+
+If you are using Power Grid Model in your research work, please consider citing our library using the following references.
+
+### Power Grid Model source code
```

Comment:
I don't think we need two heading here. Just make a big Bibtext block with two references.

Also add the same thing in readthedocs (either in index or as a separate page

Reply:
will do

in the mean time, i'm still debugging. for some reason, the zenodo publishing using the latest CITATION.cff failed. it is successful using the Github button, though.